packages/ssh-agent-unix/ssh-agent-unix.0.2.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Ssh-agent"
+description: """
+An ssh-agent protocol implementation in OCaml.
+
+For the Qubes OS unikernel using this library see [qubes-mirage-ssh-agent](https://github.com/reynir/qubes-mirage-ssh-agent)."""
+maintainer: "Reynir Björnsson <reynir@reynir.dk>"
+authors: "Reynir Björnsson <reynir@reynir.dk>"
+license: "BSD-2-clause"
+homepage: "https://github.com/reynir/ocaml-ssh-agent/"
+bug-reports: "https://github.com/reynir/ocaml-ssh-agent/issues/"
+depends: [
+  "ocaml"
+  "dune" {>= "1.0"}
+  "ssh-agent" {= version}
+  "angstrom-unix"
+  "faraday"
```

Comment:
```suggestion
  "faraday"
  "cmdliner"
```
According to this error message a dependency towards `cmdliner` is missing
```
#=== ERROR while compiling ssh-agent-unix.0.2.0 ===============================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | pinned(https://github.com/reynir/ocaml-ssh-agent/releases/download/v0.2.0/ssh-agent-v0.2.0.tbz)
# path                 ~/.opam/4.09/.opam-switch/build/ssh-agent-unix.0.2.0
# command              ~/.opam/4.09/bin/dune build -p ssh-agent-unix -j 2
# exit-code            1
# env-file             ~/.opam/log/ssh-agent-unix-23-612ba6.env
# output-file          ~/.opam/log/ssh-agent-unix-23-612ba6.out
### output ###
# File "cmd/dune", line 5, characters 60-68:
# 5 |  (libraries ssh-agent nocrypto nocrypto.unix ssh-agent-unix cmdliner))
#                                                                 ^^^^^^^^
# Error: Library "cmdliner" not found.
# Hint: try: dune external-lib-deps --missing -p ssh-agent-unix -j 2 @@default
```

Reply:
Argh! I didn't actually intend to publish ssh-agent-unix. The executable using cmdliner is just an example. I'll fix things, and do a new PR.
Sorry about that, and thank you so much for reviewing!